ground/gcs/src/plugins/uploader/tl_dfu.cpp

```diff
@@ -469,11 +472,19 @@ bool DFUObject::OpenBootloaderComs(USBPortInfo port)
         AbortOperation();
         if(!EnterDFU()) {
             TL_DFU_QXTLOG_DEBUG(QString("Could not process enterDFU command"));
+            if ((retries--) > 0) {
```

Comment:
Qt style: "Use curly braces only when the body of a conditional statement contains more than one line:"